common/src/main/java/org/dash/wallet/common/ui/dialogs/AdaptiveDialog.kt

```diff
@@ -207,11 +208,17 @@ open class AdaptiveDialog(@LayoutRes private val layout: Int): DialogFragment()
     }
 
     fun show(activity: FragmentActivity, onResult: ((Boolean?) -> Unit)? = null) {
-        onResultListener = onResult
-        show(activity.supportFragmentManager, "adaptive_dialog")
+        if (activity.lifecycle.currentState.isAtLeast(Lifecycle.State.RESUMED)) {
```

Comment:
https://console.firebase.google.com/u/1/project/dash-wallet-firebase/crashlytics/app/android:hashengineering.darkcoin.wallet/issues/2d9c86a773652972c2c748461250ce58?time=last-seven-days&types=crash&sessionEventKey=62B1C4AC030B00014852363A54355332_1690127159724001506

This should fix it in theory. Let's wait and see.